infra/k8s/sidecar.yaml

```diff
@@ -2,28 +2,35 @@ apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   name: testground-sidecar
+  annotations:
+    cni: "flannel"
 spec:
   selector:
     matchLabels:
       name: testground-sidecar
   template:
     metadata:
+      annotations:
+        cni: "flannel"
       labels:
         name: testground-sidecar
     spec:
       terminationGracePeriodSeconds: 10
       hostPID: true
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
       containers:
       - name: testground-sidecar
-        image: ipfs/testground:latest
+        image: nonsens3/testground:latest
         command: ["testground"]
-        args: ["sidecar", "--runner", "docker"]
+        args: ["--vv", "sidecar", "--runner", "k8s"]
         securityContext:
           capabilities:
-            add: ["NET_ADMIN", "SYS_ADMIN"]
+            add: ["NET_ADMIN", "SYS_ADMIN", "SYS_TIME"]
+          privileged: true
         env:
         - name: REDIS_HOST
-          value: "redis-master"
+          value: "redis-headless"
```

Comment:
Should we update our other runners?

Reply:
This is actually redundant, because even though `redis-headless` returns a PodIP, in order to resolve it - the DNS is located on the services CIDR. So we are back at the problem with the missing route (which is now added through the `initContainer`).